src/HandleRemoteObject.js

```diff
@@ -0,0 +1,58 @@
+const defaultOnError = () => {
+  const module = {
+    get: () => () => {},
+    init: () => () => {},
+  };
+  resolve(module);
+};
+
+const dynamicRemote = (remote) => {
+  return `(resolve) => {
```

Comment:
we should use webpack require.l for script loading here. i opened a PR adding webpack runtime requirements  to promise based remotes

Reply:
We should so this instead

```                     new Promise(function(resolve, reject) {
                       var __webpack_error__ = new Error();
                       if (typeof window.${
                         containerWithAtSyntax[0]
                       } !== 'undefined') return resolve();
                       var containerUrl = ${containerWithAtSyntax[1]} ? 
                       '${containerWithAtSyntax[1]}' : remoteConfig.entry
                       __webpack_require__.l(
                         containerUrl,
                         function(event) {
                           if (typeof window.${
                             containerWithAtSyntax[0]
                           } !== 'undefined') return resolve();
                           var errorType = event && (event.type === 'load' ? 'missing' : event.type);
                           var realSrc = event && event.target && event.target.src;
                           __webpack_error__.message =
                             'Loading script failed.\\n(' + errorType + ': ' + realSrc + ')';
                           __webpack_error__.name = 'ScriptExternalLoadError';
                           __webpack_error__.type = errorType;
                           __webpack_error__.request = realSrc;
                           reject(__webpack_error__);
                         },
                         '${containerWithAtSyntax[0]}',
                       );
                     })
                  ]).then(function(){
                    res(window.${containerWithAtSyntax[0]})
                  });```